packages/core/datadog-configuration.schema.json

```
@@ -0,0 +1,190 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/DataDog/dd-sdk-reactnative/blob/develop/datadog-configuration.schema.json",
+    "title": "Datadog React Native SDK Configuration",
+    "description": "Configuration for initializing the Datadog React Native SDK from the native layers.",
+    "type": "object",
+    "properties": {
+        "configuration": {
+            "description": "Configuration used for initializing the Datadog SDK.",
+            "type": "object",
+            "properties": {
+                "clientToken": {
+                    "description": "A valid Datadog client token.",
+                    "type": "string"
+                },
+                "env": {
+                    "description": "The application’s environment, for example: prod, pre-prod, staging, etc.",
+                    "type": "string"
+                },
+                "applicationId": {
+                    "description": "The RUM application ID.",
+                    "type": "string"
+                },
+                "nativeCrashReportEnabled": {
+                    "description": "Enables crash reporting for native platforms (iOS, Android). Default `false`.",
+                    "type": "boolean"
+                },
+                "nativeLongTaskThresholdMs": {
+                    "description": "The threshold for native long tasks reporting in milliseconds.",
+                    "type": "integer"
+                },
+                "longTaskThresholdMs": {
+                    "description": "The threshold for javascript long tasks reporting in milliseconds.",
+                    "type": "integer"
+                },
+                "sessionSamplingRate": {
+                    "description": "Percentage of sampled RUM sessions. Range `0`-`100`.",
+                    "type": "integer"
+                },
+                "site": {
+                    "description": "The Datadog site of your organization (can be 'US1', 'US1_FED', 'US3', 'US5', or 'EU1', default is 'US1').",
+                    "type": "string"
```

If this is an enum, why we don't use enum here?

```suggestion
                    "description": "The Datadog site of your organization (can be 'US1', 'US1_FED', 'US3', 'US5', or 'EU1', default is 'US1').",
                    "type": "string"
```

---

Yes indeed I don't think it hurts to put an enum here.

However I think changing it on the JS configuration where it's a `string` would be a breaking change, so let's only change it here.